content/techniques/security.md

```diff
@@ -82,4 +82,12 @@ app.use(
 );
 ```
 
+When there is a load balancer or reverse proxy between the server and the internet, Express may need to be configured to trust the headers set by the proxy in order to get the correct IP for the end user. To do so, first use the [Express Platform interface](https://docs.nestjs.com/first-steps#platform) when creating your `app` instance, then enable the [`trust proxy` setting](https://expressjs.com/en/guide/behind-proxies.html):
+
+ ```typescript
+const app = await NestFactory.create<NestExpressApplication>(ApplicationModule);
```

Comment:
We should use `AppModule` here for consistency.

Otherwise, looks good to me!

Reply:
Fixed